pybop/models/lithium_ion/echem.py

```diff
@@ -9,51 +9,25 @@ class SPM(EChemBaseModel):
     Wraps the Single Particle Model (SPM) for simulating lithium-ion batteries, as implemented in PyBaMM.
 
     The SPM is a simplified physics-based model that represents a lithium-ion cell using a single
-    spherical particle to simulate the behavior of the negative and positive electrodes.
+    spherical particle to simulate the behaviour of the negative and positive electrodes.
 
     Parameters
     ----------
     name : str, optional
         The name for the model instance, defaulting to "Single Particle Model".
-    parameter_set : pybamm.ParameterValues or dict, optional
-        The parameters for the model. If None, default parameters provided by PyBaMM are used.
-    geometry : dict, optional
-        The geometry definitions for the model. If None, default geometry from PyBaMM is used.
-    submesh_types : dict, optional
-        The types of submeshes to use. If None, default submesh types from PyBaMM are used.
-    var_pts : dict, optional
-        The discretization points for each variable in the model. If None, default points from PyBaMM are used.
-    spatial_methods : dict, optional
-        The spatial methods used for discretization. If None, default spatial methods from PyBaMM are used.
-    solver : pybamm.Solver, optional
-        The solver to use for simulating the model. If None, the default solver from PyBaMM is used.
-    options : dict, optional
-        A dictionary of options to customize the behavior of the PyBaMM model.
+    **model_kwargs : optional
+        Valid PyBaMM model option keys and their values.
```

Comment:
For the sake of the documentation & `help()` function, it would be good to leave the accepted **kwargs on these docstrings.